src/instructlab/model/serve_backend.py

```diff
@@ -23,6 +28,55 @@ def signal_handler(num_signal, __):
     sys.exit(0)
 
 
+def get_gpu_count() -> int:
+    # Third Party
+    import torch
+
+    if torch.cuda.is_available():
+        return torch.cuda.device_count()
```

Comment:
@RobotSail , yeah, here ^^^^

Reply:
@reidliu41 What I mean to say is that `torch.cuda.device_count` should return 0 even if cuda is not available, e.g.:

```py
In [2]: torch.cuda.is_available()
Out[2]: False

In [3]: torch.cuda.device_count()
Out[3]: 0
```

Reply:
oh thanks...is it ok for ?
```
def get_gpu_count() -> int:
    import torch
    return torch.cuda.device_count()
```